projects/auth0-angular/src/lib/auth.service.spec.ts

```diff
@@ -145,6 +166,30 @@ describe('AuthService', () => {
         done();
       });
     });
+
+    it('should return null when logged out', (done) => {
+      const claims: IdToken = {
+        __raw: 'idToken',
+        exp: 1602887231,
+        iat: 1602883631,
+        iss: 'https://example.eu.auth0.com/',
+      };
+
+      (auth0Client.isAuthenticated as jasmine.Spy).and.resolveTo(true);
+      (auth0Client.getIdTokenClaims as jasmine.Spy).and.resolveTo(claims);
+
+      service.idTokenClaims$.pipe(bufferCount(2)).subscribe((values) => {
```

Comment:
In this test I am using `bufferCount` to be able to run a test on the last 2 emitted items, to verify it changes from a idTokenClaim to null.